### ISSUE TITLE
What is the final loss size?

### ISSUE THREAD

**Issue by liming-ai** [KyleZheng1997/ReSSL#9]
Hi @KyleZheng1997,

Thanks for your contribution, could you provide the pre-training log? I really want to know the pre-train loss when training is over.

**Comment by mingkai-zheng** [comment]
Sorry, the pre-training log file has been deleted a long time ago, I could not find it right now. Would you like to tell me why you want to know the pre-training loss? Is there any reproduction issue? I would like to help you out.

**Comment by liming-ai** [comment]
> Sorry, the pre-training log file has been deleted a long time ago, I could not find it right now. Would you like to tell me why you want to know the pre-training loss? Is there any reproduction issue? I would like to help you out.

I hope to do some research in the form of relational distillation like this, I need to know the approximate size of your loss to coarsely tune the hyperparameters. I would be very grateful if you could provide it, and would appreciate your work if not. , I will find time to run your code to get reasonable results.

**Comment by mingkai-zheng** [comment]
Sorry about it, although we are not able to give the log file for ImageNet, we have provided the pre-training and evaluation log file for the CIFAR-100, please see [this issue](https://github.com/KyleZheng1997/ReSSL/issues/8#issuecomment-1011007809). We want to emphasize that the pertaining loss value will be very different if you adopt different temperatures or different memory buffer sizes, so it might not give you a good indication to the final performance. We also provided the [training code](https://drive.google.com/file/d/1j2I1Lh9Dy7cHb6YO0PZ8HXDNewXrHO-j/view) on the small datasets, start your research with CIFAR might be a quick way to verify your idea. I hope this information is helpful and good luck with your research.

**Comment by liming-ai** [comment]
> 

Thank you very much~
